src/hooks/usePosts.tsx

```diff
@@ -0,0 +1,16 @@
+import { useQuery } from 'react-query';
+
+import { getCookie } from 'utils/cookieUtils';
+import { api } from 'utils/apiUtils';
+import { PostProps } from 'types/postTypes';
+
+export function usePosts(apiQuery: string) {
```

Comment:
looks like this is not usePosts it is something like getPosts

hook should be like:

`const usePosts = () => {
  const getPosts = () => fetch('some')
  const createPost = () => fetch('some')
  const getPost = () => fetch('some')
  const deletePosts = () => fetch('some')
  
  return {
    getPosts, createPost, createPost, getPost, deletePosts
  }
}`